FFMQRLib/Enemies.cs

```diff
@@ -135,13 +150,26 @@ public class Enemy
         public byte MagicPower { get; set; }
         public byte Accuracy { get; set; }
         public byte Evasion { get; set; }
+        public int OffsetBegin { get; set; }
```

Comment:
i'm not sure what's the use of OffsetBegin/OffsetEnd here, there doesn't seem to be any reference to them and ultimately addressing should be handled by the overarching class EnemyStats (it's not right now, and i think it was a mistake on my part) and there should never be a need for anything else to be aware of it

Reply:
Some debug remainder, used for converting bank/address to file hex address. Useful to use if one is less familiar with SNES addresses. I'll remove it.